.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -27,6 +27,14 @@ If you're running self-hosted, run with `--log-level=debug` or LOG_LEVEL=debug a
 Paste the *relevant* logs here, not the entire thing and not just a link to the dashboard (others do not have permissions to view them).
 -->
 
+<details><summary>Click me to see logs</summary>
+
+```
+Copy/paste your log here
```

Comment:
Does the text `Copy/paste your log here` make it clear that you should keep the backticks (code fence)?

Or should we change the text to something more descriptive like:

```suggestion
Copy/paste your log here, between the starting and ending backticks
```